package com.alfaizkhan.usbserialcommunication;

import java.io.Closeable;

public final class SafeCloseable {
    private SafeCloseable() {}

    public static void close(Closeable closeable) {
        if (closeable != null) {
            try {
                closeable.close();
            } catch (Exception ignored) {
            }
        }
    }

    public static void close(AutoCloseable closeable) {
        if (closeable != null) {
            try {
                closeable.close();
            } catch (Exception ignored) {
            }
        }
    }
}
